FTP/src/login.java

```diff
@@ -64,23 +76,43 @@ private static void shell(FTPClient ftpClient) throws IOException {
             }else {
                 System.out.println("[" + lineSplit[0] + "] is not recognized as an internal or external command");
             }
-
         }
     }
 
-    private static void list_files_fromServer(FTPClient ftpClient){
+    private static void list_files_fromServer(FTPClient ftpClient, String remotePath){
         ftpClient.enterLocalPassiveMode();
+        System.out.println(System.getProperty("user.dir") + "\\" + remotePath + ": ");
         try{
-            FTPFile[] listFiles = ftpClient.listFiles();
+            FTPFile[] listFiles = ftpClient.listFiles(remotePath);
             if (listFiles != null) {
                 for (FTPFile file : listFiles) {
-                    System.out.print(file.getName() + '\t');
+                    if(file.isDirectory()) {
+                        System.out.print(file.getName() + "/\t");
```

Comment:
I think it's better to follow traditional "ls" mode, which put a new line after each item, instead of tabs

Reply:
I was wrong, traditional is list without new lines...